backend/app.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const bodyParser = require("body-parser");
+const passport = require("passport");
+require("dotenv").config();
+
+// const users = require("./routes/user");
+const bounty = require('./routes/bounty');
+
+mongoose.connect(process.env.MONGO_DB_LOCAL, { useNewUrlParser: true }).then(
+    () => {console.log('Database is connected') },
+    err => { console.log('Can not connect to the database' + err)}
+);
+
+const app = express();
+// app.use(passport.initialize());
+// require('./passport')(passport);
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.get("/", function(req, res) {
+    res.send("Hello");
```

Comment:
is this really what we went to send back the response?

Reply:
I think it's a nice response, but the express server isn't being used in final for routing, so we could just send back nothing as a response or a page not found page, because that's only accessed through the backend port which is 3000 while front end is 5000 and they use a proxy to make api calls easy.